roofit/histfactory/src/ParamHistFunc.cxx

```diff
@@ -150,34 +150,32 @@ ParamHistFunc::ParamHistFunc(const char* name, const char* title,
   // Add the parameters (with checking)
   addVarSet( vars );
   addParamSet( paramSet );
- 
 }
 
 
 Int_t ParamHistFunc::GetNumBins( const RooArgSet& vars ) {
-  
+
   // A helper method to get the number of bins
-  
+
   if( vars.getSize() == 0 ) return 0;
-    
+
   Int_t numBins = 1;
 
   RooFIter varIter = vars.fwdIterator() ;
   RooAbsArg* comp ;
   while((comp = (RooAbsArg*) varIter.next())) {
```

Comment:
If you are at it anyway:
```c++
for (auto arg : vars) {
...
```
Those loops are ~20% faster (which doesn't make a difference here), but well. 🙂